migrating/ossm-migrating-premigration-checklist-assembly.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-read-me-assembly.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklist-assembly]
+= Migrating from 2.6.z to 3.0 premigration checklist
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-migration-guides-assembly
+
+toc::[]
+
+Before you start migrating your `controlplane` from {SMProductName} 2.6 to 3.0, do the following:
+
+* [ ] Read "Important information to know if you are migrating from OpenShift Service Mesh 2.6".
+* [ ] Upgrade the {SMProduct} Operator to the latest 2.6.z release.
+* [ ] Upgrade your `ServiceMeshControlPlane` version to the latest {SMProduct} 2.6.z release.
+
+== Disable addons on your `ServiceMeshControlPlane`
+
+These addons are not supported in {SMProduct} 3.0 and must be disabled before migrating.
+
+** [ ] Disable network policy management by setting `spec.security.manageNetworkPolicy=false`. If you wish to keep the Network Policies created by the 2.6 ServiceMeshControlPlane, you will need to recreate and manage these manually.
+** [ ] Disable Prometheus by setting `spec.addons.prometheus.enabled=false`. See "User workload monitoring" for instructions configuring {SMProduct}h 3.0 with OpenShift Monitoring as a replacement for Prometheus.
+** [ ] Disable Kiali by setting `spec.addons.kiali.enabled=false``. See "Kiali Operator provided by Red Hat" for instructions installing and configuring {KialiProduct} with {SMProduct} 3.0.
+** [ ] Disable Grafana by setting `spec.addons.grafana.enabled=false`.
+** [ ] Disable {DTProductName} by setting `spec.tracing.type=None`. See "{DTProductName}" for instructions on configuring {SMProduct} 3.0 with {DTProductName}.
+IOR is disabled.
```

Comment:
For the last two steps, "IOR is disabled" and "Disable default ingress and egress gateways", we should link to the specific content that describes how to make these changes. That is, [Route migration](https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/ossm-route-migration.html) and [Gateway Migration](https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/ossm-gateway-migration.html)